Automattic/jetpack
# Boost: Make regenerate button always visible, but disable when generating

Fixes #35293

## Proposed changes:

* Updates the Cloud CSS status section to always show the regenerate button, but disable it when generation is running;
* Hovering over the regenerate button changes the cursor "not allowed", to further indicate that the action is not allowed.

	<details><summary>Before</summary>
	
	![CleanShot 2024-01-29 at 19 44 49](https://github.com/Automattic/jetpack/assets/11799079/2b32ead3-0ea6-46ab-89f8-dfa49536f969)
	
	
	</details>
	
	<details><summary>Now</summary>
	
	![CleanShot 2024-01-29 at 19 45 25](https://github.com/Automattic/jetpack/assets/11799079/603123c7-8d08-4022-99a6-8de98fbad4af)
	
	
	</details>



### Other information:


## Jetpack product discussion



pedxs5-mg-p2#comment-223

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost (premium);
* Make sure Cloud CSS generation works and button UI behaves as expected.
